### DISCUSSION TITLE
Audio Files

### DISCUSSION THREAD

**Discussion by mertduden** [vzhd1701/gridplayer#36] [General]
First of all, great work thank you so much,

I wonder that is there any way to play audio file with videos simultaneously ?

**Comment by vzhd1701** [comment]
I am sorry, but I am not sure it will be very useful. Could you please describe the use case scenario? Why would you need to play audio files alongside videos?

**Reply by mertduden** [reply]
I have three media that are 2 videos and 1 audio all recorded in same interval. Records are separately, but listeners want to listen / watch them at the same time to control the interactions in all manner. It would be very beneficial in my use-case.

**Reply by aromapunk** [reply]
Thank you for your great work!
that would be very useful for me!
we are a group running a live stream from different locations at the same time. Because this function is not included, I always have to open an extra window that is open in parallel and the sound is either ON or OFF...
I would like to be able to turn on the sound in at least one window. please take a look at the channel

https://odysee.com/@Snickerpie:b/PeoplesConvoy:a

I have no idea what kind of tool that is, the operators say it's homemade.
greetings from Austria
AromaPunk

**Reply by aromapunk** [reply]
i woll not play audio files alongside the video... i will have a sound ON / OFF function of the windows...
